ATTACK/Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    private GameObject[] spawnPointsT1;
+    private GameObject[] spawnPointsT2;
+
+    public GameObject character;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        spawnPointsT1 = GameObject.FindGameObjectsWithTag("Team1Spawn");
+        spawnPointsT2 = GameObject.FindGameObjectsWithTag("Team2Spawn");
+
+        for (int i = 0; i < 3; i++)
+        {
+            SpawnCharacter(character, spawnPointsT1, i, 0);
+        }
+
+        for (int i = 0; i < 3; i++)
+        {
+            SpawnCharacter(character, spawnPointsT2, i, 1);
+        }
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    private void SpawnCharacter(GameObject character, GameObject[] teamSpawns, int spawnIndex, int teamNumber)
+    {
+        Vector3 spawnPoint = teamSpawns[spawnIndex].transform.position;
+        
+        // Quaternions are required to Instantiate at a Vec3. Alternative will require a transform which makes the characters a child of their spawnpoint

```

Comment:
Not sure this comment is correct. If you copy the transform or copy the translation and rotation of the spawn point, they should not become children of the game object.

Also teamNumber * 180 may introduce an error, as we don’t know which team will be on which side, or even if the game field will be rotated that way. Better would be to rotate them relative to the game field.

Reply:
I think rotation towards a point and projecting onto a certain axis is more work than necessary, we do know what team will be on what side, it's only a matter of how the camera is rotated/positioned. Generally, if order is followed, then team 1 is on the left and team 2 on the right, which is currently true.

There are some things that can make this cleaner, however, like adding enums for teams instead of magic numbers

Reply:
Or we could just rotate the spawn points manually and make the characters children of those, which is easier in every way and I don think that parent-child relation will cause any problem, but we can think of that later, it works and doesn't really matter right now